sdk/src/asset_handlers/tiff_io.rs

```diff
@@ -1301,7 +1302,7 @@ fn tiff_clone_with_tags<R: Read + Seek + ?Sized, W: Read + Write + Seek + ?Sized
     }
 
     tc.clone_tiff(&mut tiff_tree, page_0, asset_reader)?;
-
+    bo.flush()?;
```

Comment:
I would just the flush to the clone_tiff function so that all calls are covered.